tests/test_smile.py

```diff
@@ -643,7 +643,8 @@ async def test_connect_legacy_anna(self):
 
         await self.device_test(smile, testdata)
         _LOGGER.info(" # Assert master thermostat")
-        assert smile._sm_thermostat  # pylint: disable=protected-access
+        assert smile._is_thermostat  # pylint: disable=protected-access
```

Comment:
For tests, it is not uncommon to put

`# pylint: disable=protected-access`

In top of the file (so it applies the whole module)

Reply:
Tnx!